build.gradle

```diff
@@ -49,7 +49,9 @@ dependencies {
                     "com.sun.jersey:jersey-core:1.19",
                     "com.sun.jersey.contribs:jersey-multipart:1.19",
                     "org.apache.taglibs:taglibs-standard-impl:1.2.5",
-                    "org.jsoup:jsoup:1.9.2"
+                    "org.jsoup:jsoup:1.9.2",
+                    "org.apache.poi:poi:3.9",
```

Comment:
Is there any particular reason why these versions are chosen. Latest stable version appears to be 3.14?